x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/RollupJobConfig.java

```diff
@@ -47,6 +53,9 @@
     private static final String PAGE_SIZE = "page_size";
     private static final String INDEX_PATTERN = "index_pattern";
     private static final String ROLLUP_INDEX = "rollup_index";
+    private static final List<String> DEFAULT_HISTO_METRICS = Arrays.asList(MaxAggregationBuilder.NAME,
+        MinAggregationBuilder.NAME,
+        ValueCountAggregationBuilder.NAME);
```

Comment:
Do we need `value_count` as a default?  Both histo's naturally save their doc_count (since it's part of the return data of a histo/date_histo agg).  Is there a specific need for value counts instead of just doc count?

Reply:
Probably not, I can remove it.

Reply:
Oops sorry, this probably should have a top-level comment in the ticket, not on a source line.  I just meant in general, do we need to default to storing value_count? Min/max make sense, i'm less sold on value count (since we already have doc count)

@colings86 wdyt?

Reply:
I agree I think the majority of use cases will care about doc count rather than value count so if a user explicitly wants value count I think they can use the override mechanism to add it instead of use storing this extra information

Reply:
Alright, I will pull `value_count` out of our default metrics. Will update the PR shortly :)